kpack-image-builder/config/manager/kustomization.yaml

```diff
@@ -13,4 +13,7 @@ kind: Kustomization
 images:
 - name: cloudfoundry/korifi-kpack-image-builder
   newName: korifi-kpack-image-builder
-  newTag: latest
+  newTag: C2297A37-57DE-4D45-ADB4-E2BBE5312282
```

Comment:
What is this change?

Reply:
Oh--this didn't need to be in here. It's just a change that gets applied each time you build.